ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,21 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-disconnected-private-provision
+  - ref: workers-rhel-aws-provision
+  - ref: workers-rhel-pre-hook-sshtunnel
+  - chain: workers-rhel
+  env:
+  - name: USE_INTERNAL_SSH_BASTION
```

Comment:
How about renaming it to `USE_SSH_BASTION_POD`?

Reply:
addressed in https://github.com/openshift/release/pull/29782#discussion_r909222081